synapse/config/push.py

```diff
@@ -33,20 +26,20 @@ def read_config(self, config):
         # 'email'section'. Check for the flag in the 'push' section, and log,
         # but do not honour it to avoid nasty surprises when people upgrade.
         if push_config.get("redact_content") is not None:
-            reactor.callWhenRunning(lambda: logger.warn(
+            print(
```

Comment:
why not just `logger.warn(<msg>)` instead of the previous construct 
(`reactor.callWhenRunning(lambda: logger.warn(<msg>`)?
Shouldn't that be faster as well with the functionality that this message will be part of homeserver.log?
This is especially useful in the case when you run it as service and won't see any console-output

Reply:
At this point the logging hasn't been set up and we've not yet daemonized, and using print is consistent with where we do it elsewhere

Reply:
All right. Good to know